### PR TITLE
tracked_bad_severity: add a comment to explain

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

File: auto_nag/scripts/tracked_bad_severity.py [hunk 1]
```diff
@@ -59,7 +55,12 @@ def get_bz_params(self, date):
         return params
 
     def get_autofix_change(self):
-        return {"severity": "major"}
+        return {
+            "comment": {
+                "body": f"This bug is tracked but with a small severity so change it to major.\n{self.get_documentation()}"
```
Comment (sylvestre) [inline auto_nag/scripts/tracked_bad_severity.py]:
```suggestion
                "body": f"This bug is tracked by a release manager but with a small severity so change it to major.\n{self.get_documentation()}"
```

Comment (calixteman) [reply auto_nag/scripts/tracked_bad_severity.py]: For this tool we don't check if a release manager changed the flag (even if it is very likely).

Comment (sylvestre) [reply auto_nag/scripts/tracked_bad_severity.py]: the tracking action is done only by release manager

Comment (sylvestre) [reply auto_nag/scripts/tracked_bad_severity.py]: @calixteman ping?

**Pull request merged**